src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -200,39 +201,44 @@ public async Task RetrieveCameraDevice(bool force = false)
 			IsBusy = true;
 			cameraId = GetCameraId();
 
-			if (string.IsNullOrEmpty(cameraId))
+			if (cameraId == null || string.IsNullOrEmpty(cameraId))
```

Comment:
We already did the null check on IsNullOrEmpty method.

Reply:
Same as [this comment](https://github.com/xamarin/XamarinCommunityToolkit/pull/1009#discussion_r588798396), unless we include `cameraId == null`, Roslyn assumes that `cameraId` could still be `null`, despite us using `string.IsNullOrEmpty`